mbof/views.py

```diff
@@ -39,6 +39,25 @@ def get_queryset(self):
         return Event.objects.filter(startTime__lte=current_time, endTime__gte=current_time).order_by('postingTime')
 
 
+class CurrentEventViewSet(EventViewSet):
```

Comment:
`CurrentEventViewSet` was already added in an earlier PR.  Why is it here again now?